### ISSUE TITLE
convenience imports in __init__

### ISSUE THREAD

**Issue by ryanjulian** [ryanjulian/rllab#36]

**Comment by ryanjulian** [comment]
e.g. rather than
```python
from sandbox.rocky.tf.policies.gaussian_mlp_policy import GaussianMLPPolicy
```
Add package-level imports to `__init__.py` so we can do
```python
from sandbox.rocky.tf.policies import GaussianMLPPolicy
```

**Comment by ryanjulian** [comment]
The change should also rewrite all of the imports across the codebase to reflect the new style.

**Comment by ryanjulian** [comment]
Fixed in #66
